content/sphinx.md

```diff
@@ -323,42 +325,36 @@ the list of extensions in `conf.py`:
 extensions = ['sphinx.ext.mathjax']
 ```
 
+**Add here: https://myst-parser.readthedocs.io/en/latest/syntax/optional.html#syntax-amsmath**
+
 The following shows how to inline mathematics within a text:
 
-```rst
-This is an inline equation embedded :math:`a^2 + b^2 = c^2` in text.
+```md
+This is an inline equation embedded {math}`a^2 + b^2 = c^2` in text.
 ```
 
-An equation and equation array:
-```rst
-.. math::
-  :label: myequation
+An equation:
 
-  a^2 + b^2 = c^2
+:math:`a^2 + b^2 = c^2`
 
+or:
 .. math::
```

Comment:
~~~
```{math}

Reply:
This still needs to be fixed (but I can merge and create anoher PR)